editor/components/FatalityUnitsCards.tsx

```diff
@@ -0,0 +1,173 @@
+import { Crash } from "@/types/crashes";
+import { Card, ListGroupItem, Image } from "react-bootstrap";
+import { getInjuryColorClass } from "@/utils/people";
+
+interface FatalityUnitsCardsProps {
+  crash: Crash;
+}
+
+/**
+ * A section of the Fatality Details page that renders a unit card for every
+ * unit in the crash that has a fatality, contributing factor, or charge associated with it
+ */
+export default function FatalityUnitsCards({ crash }: FatalityUnitsCardsProps) {
+  const victims = crash.people_list_view?.filter(
+    (person) => person.prsn_injry_sev_id === 4
+  );
+  const units = crash.units;
+
+  const charges = crash.charges_cris;
+
+  const BASE_PATH = process.env.NEXT_PUBLIC_BASE_PATH || "";
+
+  return (
+    <div>
+      {units?.map((unit) => {
+        // Get the list of fatalities in the unit
+        const unitVictims = victims?.filter(
+          (victim) => victim.unit_nbr === unit.unit_nbr
+        );
```

Comment:
It would be cool if this unit-person hierarchy came through from the graphql API. any reason not to pull the people records via the unit-person relationship instead?